src/operator/api/index.ts

```diff
@@ -214,64 +163,20 @@ export const getOrgs = (
 export const getOrgById = (_id: string): ReturnType<typeof getOrg> => {
   const organization: Organization = {
     id: '123',
-    idpeID: '123',
+    quartzId: '123',
     name: 'Best Org',
     region: 'us-west',
     provider: 'Zuora',
     date: '01/01/2010',
-    account: {
-      id: '123',
-      marketplace: null,
-      balance: 0,
-      billingContact: {
-        companyName: 'Influx',
-        email: 'asalem@influxdata.com',
-        firstName: 'Ariel',
-        lastName: 'Salem',
-        country: 'USA',
-        street1: '123 Main St',
-        city: 'New York',
-        subdivision: 'NY',
-        postalCode: 30000,
-      },
-      users: [],
-      type: 'pay_as_you_go',
-    },
+    accountType: 'pay_as_you_go',
+    accountBalance: 10,
+    accountId: 'pay123',
+    accountEmail: 'paid@account.com',
   }
 
   return makeResponse(200, organization)
 }
 
-export const getMe = (): ReturnType<typeof apiGetMe> => {
```

Comment:
Moved this to the `me` reducer since we're likely to use some of the meta data provided in that request when conditionally rendering the upgrade now button